packages/api-client/src/views/Request/libs/watch-mode.ts

```diff
@@ -126,6 +126,7 @@ const unwrapSchema = (schema: ZodSchema): ZodSchema => {
   if (schema instanceof z.ZodOptional) return unwrapSchema(schema.unwrap())
   if (schema instanceof z.ZodDefault) return unwrapSchema(schema._def.innerType)
   if (schema instanceof z.ZodEffects) return unwrapSchema(schema._def.schema)
+  if (schema instanceof z.ZodCatch) return unwrapSchema(schema._def.innerType)
```

Comment:
@hanspagel and the zod traverser gets even better, maybe we should pull this out of watch mode and into its own lib. Could be useful to others